### ISSUE TITLE
[Accessibility] Localization of "End of Posts" string in feed.component.html

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4981]
# Summary

## Describe the bug
The string "End of Posts" on line 290 in the `feed.component.html` file needs to be localized to support multiple languages.

## To Reproduce
1. Navigate to the `apps/web-mzima-client/src/app/feed/feed.component.html` file.
2. Locate the following code block on line 290:
```html
<div class="endofPosts" *ngIf="!loadingMorePosts && postCurrentLength >= total">
  End of Posts
</div>
```

## Expected behavior
The "End of Posts" string should be localized and retrieved from a translation file or service, so that it can be displayed in the user's preferred language.

## How to solve
1. In the `feed.component.html` file, replace the hardcoded "End of Posts" string with a translation key ( something like., `{{ 'FEED.END_OF_POSTS' | translate }}`)

## Screenshots
<img width="1200" alt="Screenshot 2024-07-24 at 09 42 44" src="https://github.com/user-attachments/assets/f081033e-dd68-40be-88dd-4294d49dc2aa">

**Comment by Angamanga** [comment]
Closing but keeping in Linear: https://linear.app/ushahidi/issue/USH-1476/[accessibility]-localization-of-end-of-posts-string-in
